Documentation/General/emergentArchitectureDesign/emergentArchitectureDesign.tex

```diff
@@ -118,11 +118,15 @@ \subsubsection{REST API}
 
 \subsubsection{Music matching algorithm}\label{MatchingAlgorithm}
 The music matching algorithm determines and compares moods for songs, rooms and users.
-Moodcat will be developed around a learning algorithm.
-This means that Moodcat will improve over time.
-Default values will be determined by examining the audio features from our data set.
-Then, through a system of user votes, the system will learn and improve over time.
+The classification of songs will be done due to classification by the user, this will be done with gamification.
+Therefore Moodcat will improve over time, because the more people classify the more accurate it gets.
+Default values will be set to 0 for valence and arousal. We express them in a range between -1 and 1 and so we choose the centre to begin with.
 
+We a user does a down-votes it will adjust the vector to the classification the user fills in after down-voting.
```

Comment:
> ~~We a user does a down-votes it will adjust the vector to the classification the user fills in after down-voting.~~ When a user votes down, the user will be requested to classify the song.